packages/bot/src/stores/flyout-store.js

```diff
@@ -155,39 +162,39 @@ export default class FlyoutStore {
                 current_block_index = index;
             }
         });
-        const last_position = Object.keys(xml_list_group).length - 1;
-
-        const adjustIndexOutOfBounds = (current_index, last_index) => {
-            let current_pos = current_index;
-            if (current_pos < 0) {
-                current_pos = last_index;
-            } else if (current_pos > last_index) {
-                current_pos = 0;
-            }
 
-            return current_pos;
+        const getNextBlock = async (xml, currentIndex, direction) => {
+            const nextIndex = currentIndex + (direction ? 1 : -1);
```

Comment:
Please use snake_case for variables that are not functions.

Reply:
done